esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -6047,13 +6091,11 @@ void CWsDfuEx::dFUFileAccessCommon(IEspContext &context, const CDfsLogicalFileNa
         groups.push_back(clusterGroup);
     }
 
-    StringBuffer keyPairName;
-    unsigned port;
-    bool secure;
     getFileDafilesrvConfiguration(keyPairName, port, secure, fileName, groups);
-#ifdef _USE_OPENSSL
+# ifdef _USE_OPENSSL
```

Comment:
Is this deliberate?  Normally we would indent the #

Reply:
it was deliberate, I took inspiration from somewhere else I saw do in our code base (and our recent helm chart indentation).
I think helps spot/show nesting.

Reply:
>Normally we would indent the #

oh ok, not sure I've seen that - but happy to change to that.

Reply:
changed